ui/src/containers/Capabilities/index.js

```diff
@@ -14,47 +14,59 @@ import { StyledCapabilities } from "./style.css";
 import BeatLoader from "react-spinners/BeatLoader";
 import { useEventCallback } from "rxjs-hooks";
 
+import Url from "url-parse";
+
 const code = (browser = "UNKNOWN", version = "", origin = "http://selenoid-uri:4444") => {
+    const url = new Url(origin);
     return {
         yaml: `# selenium: "${origin}"
 # please note that real accessible selenoid uri can be different        
 browserName: "${browser}"
-version: "${version}"
+browserVersion: "${version}"
 enableVNC: true
 enableVideo: false 
 `,
         curl: `curl -X POST '${origin}/wd/hub/session' -d '{ 
             "desiredCapabilities":{
                 "browserName":"${browser}", 
-                "version": "${version}", 
-                "platform":"ANY",
-                "enableVNC": true,
-                "name": "this.test.is.launched.by.curl",
-                "sessionTimeout": "120s"
+                "browserVersion": "${version}", 
+                "platformName":"ANY",
+                "selenoid:options": {
+                    "enableVNC": true,
+                    "name": "this.test.is.launched.by.curl",
+                    "sessionTimeout": "120s"
+                }
             }
         }'
 `,
         java: `DesiredCapabilities capabilities = new DesiredCapabilities();
-capabilities.setBrowserName("${browser}");
-capabilities.setVersion("${version}");
-capabilities.setCapability("enableVNC", true);
-capabilities.setCapability("enableVideo", false);
-
+capabilities.setCapability("browserName", "${browser}");
+capabilities.setCapability("browserVersion", "${version}");
+capabilities.setCapability("selenoid:options", new HashMap<String, Object>(){
```

Comment:
maybe java11 with Map.of?

Reply:
In case of Map.of inline call we would anyway need to specify type parameters because otherwise Java will not be able to infer them.
```
Map.of<String, Object>("browserName", true, "browserVersion", true)
```

Reply:
that's anyway more compact

Reply:
shouldn't be Map.<String, Object>of?

Reply:
Ok, updated.